src/duplicacy_gcdstorage.go

```diff
@@ -328,8 +358,9 @@ func CreateGCDStorage(tokenFile string, storagePath string, threads int) (storag
 
 	storage.isConnected = true
 
+	storage.DerivedStorage = storage
+	storage.SetDefaultNestingLevels([]int{0}, 0)
```

Comment:
I don't understand most of what you did here so this question may be stupid, but why here the default nesting levels is 0 and 0, instead of the "default" 1 and 1? Note that i also saw this nesting for other storages unrelated to google: s3cstorage, or azurestorage.

Reply:
This is the backward-compatible default, which could be either 2,3 and 2, or 0 and 0, depending on the storage type, so it has to be set in the constructor.  The new "default" 1 and 1 is the same for all storage, so it can be set in SetNestingLevels.

Reply:
In that case, it is too much to make a note of this somewhere? Let's say maybe in the class (is it a class?) StorageBase?